tests/functional/rbac/checkers.py

```diff
@@ -0,0 +1,183 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Checker for checking if performing some actions is truly denied"""
+
+import json
+import re
+from functools import partial
+from typing import Type, Collection, Union, Callable, Optional
+from urllib import parse
+
+import allure
+import requests
+from adcm_client.base import ObjectNotFound
+from adcm_client.objects import (
+    ADCMClient,
+    Cluster,
+    Service,
+    Component,
+    Provider,
+    Host,
+    User,
+    Group,
+    Role,
+    Policy,
+    Bundle,
+    ADCM,
+)
+
+from tests.library.consts import HTTPMethod
+from tests.functional.tools import AnyADCMObject, AnyRBACObject
+
+RoleTargetObject = Union[AnyADCMObject, AnyRBACObject, Bundle, ADCM]
+RoleTargetType = Type[RoleTargetObject]
+
+
+class ForbiddenCallChecker:  # pylint: disable=too-few-public-methods
```

Comment:
need docstrings to class and method to describe this

Reply:
Added docstring to `__call__`, not sure where there's a need for more documentation